### ISSUE TITLE
Inability to Complete New Job/Validation Failure in Dart

### ISSUE THREAD

**Issue by StudentUserMattC** [APTrust/dart#557]
When creating a new job, DART would allow me to get all the way to the penultimate screen that says 'Upload Targets' and then, when I would click 'next', nothing would happen.  Upon returning to the home screen, there would be a message on the right side of the screen (under 'recent jobs') that said 'validation failed'.   For context, I use a PC with (I believe) Windows 11.

**Comment by diamondap** [comment]
Can you send your log file to help@aptrust.org? If you click on Help > About, you'll see a dialog showing where the log file is on your local machine. If you zip that up and send it to help@aptrust.org, I can take a look. 

You could also upload the log as an attachment to this issue report, but if you go that route, please make sure the log file doesn't contain any sensitive information.

**Comment by StudentUserMattC** [comment]
Absolutely!  I can send that over right now.  I can definitely send it to
***@***.***  Thank you!

Best,

Matt Cavanagh
